.github/workflows/jit.yml

```diff
@@ -93,12 +93,14 @@ jobs:
             compiler: clang
           - target: aarch64-unknown-linux-gnu/gcc
             architecture: aarch64
-            runner: ubuntu-22.04
```

Comment:
I wonder if it's worth trying to bump all jobs to 24.04 at the same time. I downgraded us a couple of weeks back because there were issues with the runner images, but maybe we could see if that's resolved now.